search-sentiments-map/public/script.js

```diff
@@ -23,6 +23,9 @@ const SEE_MORE_OR_LESS_ID = 'see-more-or-less';
 const SEE_MORE = 'See More';
 const SEE_LESS = 'See Less';
```

Comment:
Rename: TOGGLE_SEE_MORE, TOGGLE_SEE_LESS

Also, i think show more and show less would make more sense (name and text)

Reply:
okk